src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,57 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int QUANTITY_INDEX = 1;
+    private static final int ACTION_INDEX = 0;
+    private static final String SUPPLY_ACTION = "supply";
+    private static final String BUY_ACTION = "buy";
+    private int supply = 0;
+    private int buy = 0;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        supply = 0;
+        buy = 0;
```

Comment:
they should be method-level variables
```suggestion

    public void getStatistic(String fromFileName, String toFileName) {
```